src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -168,6 +176,69 @@ private IEnumerable<CompletionItem> GetResourceTypeCompletions(SemanticModel mod
                 .ToList();
         }
 
+        private IEnumerable<CompletionItem> GetModulePathCompletions(SemanticModel model, BicepCompletionContext context)
+        {
+            if (!context.Kind.HasFlag(BicepCompletionContextKind.ModulePath)
+            || FileResolver.TryResolveModulePath(model.SyntaxTree.FileUri, ".")?.LocalPath is not string cwd
+            || new Uri(cwd) is not {} cwdUri
```

Comment:
This is an odd way to assign a variable and complicates reading of the if check - I'd just stick with `var cwdUri = new Uri(cwd);` after the if block.